src/applications/mhv-supply-reordering/components/alerts/AlertSomethingWentWrong.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import DlcEmailLink from '../DlcEmailLink';
+import DlcTelephoneLink from '../DlcTelephoneLink';
+
+const AlertSomethingWentWrong = () => (
+  <va-alert status="error" data-testid="reorder-alert--something-went-wrong">
+    <h3 slot="headline">We’re sorry. Something went wrong on our end.</h3>
+    <div className="mdot-server-error-alert">
+      <p>
+        You can’t place an order for hearing aid or CPAP supplies because
+        something went wrong on our end.
+      </p>
+      <p className="vads-u-font-weight--bold vads-u-margin-y--1 vads-u-font-family--serif">
+        What you can do
```

Comment:
Should this be a heading? Genuinely not sure.

Reply:
Couldn't hurt to make this an h4, but I might hold off changing it until an a11y review.